source/includes/resources/deposit/_representation.md

```diff
@@ -24,46 +24,51 @@ The Deposition resource is used for uploading and editing records on Zenodo.
+| `partof_pages`</br>*string*                  | No                                                   | Pages numbers of book                                        |
+| `thesis_supervisors`</br>*array of objects*  | No                                                   | Supervisors of the thesis. Same format as for `creators`.    |
+| `thesis_university`</br>*string*             | No                                                   | Awarding university of thesis.                               |
+| `subjects`</br>*array of objects*            | No                                                   | Specify subjects from a taxonomy or controlled vocabulary. Each term must be uniquely identified (e.g. a URL). For free form text, use the keywords field. Each array element is an object with the attributes:</br>* `term`: Term from taxonomy or controlled vocabulary.</br>* `identifier`: Unique identifier for term.</br>* `scheme`: Persistent identifier scheme for id (automatically detected).</br></br>**Example:** `[{"term": "Astronomy", "identifier": "http://id.loc.gov/authorities/subjects/sh85009003", "scheme": "url"}]` |
+| `version`</br>*string*                       | No                                                   | Version of the resource. Any string will be accepted, however the suggested format is a semantically versioned tag (see more details on semantic versioning at [semver.org](http://semver.org/))</br>**Example:** `2.1.5` |
+| `language`</br>*string*                      | No                                                   | Specify the main language of the record as ISO 639-2 or 639-3 code, see [Library of Congress ISO 639 codes list](https://www.loc.gov/standards/iso639-2/php/code_list.php).</br>**Example:** `eng` |
+| `location`</br>*array of objects*            | No                                                   | List of locations</br>* `lat (double):` latitude </br>* `lon (double):` longitude</br>* `place (string):` place's name </br>* `description (string):` place's description (optional)</br>**Example:** `[{"lat": 34.02577, "lon": -118.7804, "place": "Los Angeles"},{"lat": -33.85249, "lon": 151.21046, "place": "Sydney"},{"lat": 45.90621, "lon": 6.1267, "place": "Annecy"}]` |
+| `dates`</br>*array of objects*               | No                                                   | List of dates</br>* `start (ISO date string):` start date (\*) </br>* `end (ISO date string):` end date (\*)</br>* `type (Collected, Valid, Withdrawn):` The interval’s type (required) </br>* `description (string):` The interval’s description (optional)</br>(\*) Note that it is possible to indicate only the start date or the end date, but you must specify at least one of them.</br>**Example:** `[{"start": "2018-03-21", "end": "2018-03-21", "type": "Collected","description": "Sample collection."}]` |
```

Comment:
I would also add this: "In order to have a one-day-interval, `start_date` and `end_date` should have the same value."

Reply:
Instead of "list of dates" I think we should say "list of date intervals", since each object of the list is an interval defined by a start and an end date.